README.md

```diff
@@ -2,26 +2,30 @@
 
 <img src="https://user-images.githubusercontent.com/13848158/97081166-8f568800-1611-11eb-991c-e9bc1344074e.png" height="95" />
 
-**A super-easy way to record, search and compare AI experiments.**
+**A super-easy way to record, search and compare 1000s of ML training runs**
```

Comment:
Suggested change:

Effortlessly record, search and compare thousands of machine learning training runs.

OR

We offer convenient solution for recording, search and comparison of thousands of machine learning training runs.